source/CAS/Client.php

```diff
@@ -2626,6 +2622,25 @@ private function _callback()
         throw new CAS_GracefullTerminationException();
     }
 
+    /**
+     * Check if application/xml or text/xml is pressent in HTTP_ACCEPT header values
+     * when return value is complex and contains attached q parameters.
+     * Example:  HTTP_ACCEPT = text/html,application/xhtml+xml,application/xml;q=0.9
+     * @return bool
+     */
+    private function isXmlResponse()
+    {
+        if (!array_key_exists('HTTP_ACCEPT', $_SERVER)) {
+            return false;
+        }
+        if (str_replace('application/xml', '', $_SERVER['HTTP_ACCEPT']) != $_SERVER['HTTP_ACCEPT']) {
```

Comment:
I did notice that you fix the str_replace compare value. However as I mentioned before, `strpos` should be better (in meaning) if you want to check whether your Accept header contains a string, since you don't really need to use the string replacement result (for output). Or, do you have any performance reference so that you are preferring `str_replace`?

I though this would be better with a unit test but I guess I will just give up and probably do it for you later.

Reply:
as requested the change was made and str_replace check was changed to use strpos.

Best regards